src/components/ViewServiceDetails/TotalAmount/TotalAmount.js

```diff
@@ -0,0 +1,33 @@
+import React, { Fragment } from 'react';
+import styles from './TotalAmount.css';
+
+const getTotalAmount = data => {
+    let totalAmount = 0;
+    data.forEach(service => {
+        const amount = Number(service.amount);
+        if (!isNaN(amount)) {
+            totalAmount += amount;
+        }
+    });
+    return totalAmount.toLocaleString('en-IN', {
+        maximumFractionDigits: 2,
+        style: 'currency',
+        currency: 'INR',
+    });
+};
+
+const TotalAmount = ({ data }) => {
+    return (
+        <Fragment>
```

Comment:
Fragment is not really needed here. 

Also, following is more readable, right?

```
if( !data || data.length === 0) return null;
return(
<div className={styles['amount-cmp']}>
    Total Amount : {getTotalAmount(data)}
</div>
)
```

Reply:
done